run-it/single-tenant-host/README.md

```diff
@@ -31,12 +33,12 @@ This will run `pre-deploy` and thus it will recreate the `web-server` and the `g
 
 Some tools are available to debug the databases is needed:
 
-- pgadmin
-- redis-commander
+- [pgadmin](https://www.pgadmin.org/)
+- [redis-commander](https://github.com/joeferner/redis-commander)
 
 To operate them use `docker compose up --profile databases --profile database-debug`.
 
 ## Running the whole stack
 
 - Start the whole stack: `docker compose --profile databases --profile parabol --profile database-debug --profile chronos up -d`.
```

Comment:
no need to list all the profiles if you're running the whole stack! `docker compose up -d`

Reply:
Mmmm I'll check that but you might be right!

Reply:
It doesn't work without adding services 
```
$ docker compose up -d
no service selected
```

That only works to run components without a service, and in this stack all components have services. Keeping those instructions then.